docs/content/configuration/csi_ug_config_create_statefulset.md

```diff
@@ -1,127 +1,126 @@
+            volumeMounts:
+              - name: demo-volume-file-system
+                mountPath: "/data"
+          volumes:
           - name: demo-volume-file-system
-            mountPath: "/data"</b>
-      volumes:
-      - name: demo-volume-file-system
-        persistentVolumeClaim:
-          claimName: demo-pvc-file-system
-</pre>
+            persistentVolumeClaim:
+              claimName: demo-pvc-file-system
 
 ## Creating a StatefulSet with raw block volume
 
-Create a StatefulSet yaml file, similar to the following demo-statefulset-raw-block.yaml file.
-
-<pre>
-kind: StatefulSet
-apiVersion: apps/v1
-metadata:
-  name: demo-statefulset-raw-block
-spec:
-  selector:
-    matchLabels:
-      app: demo-statefulset
-  serviceName: demo-statefulset
-  replicas: 1
-  template:
+Create a StatefulSet YAML file, similar to the following `demo-statefulset-raw-block.yaml` file.
+
+Be sure to indicate the `volumeDevices`, listing each volumes name and path. In this example, the `devicePath` is listed as `"/dev/block"`.
+
+    kind: StatefulSet
+    apiVersion: apps/v1
     metadata:
-      labels:
-        app: demo-statefulset
+      name: demo-statefulset-raw-block
     spec:
-      containers:
-      - name: demo-container
-        image: registry.access.redhat.com/ubi8/ubi:latest
-        command: [ "/bin/sh", "-c", "--" ]
-        args: [ "while true; do sleep 30; done;" ]
-        <b>volumeDevices:
+      selector:
+        matchLabels:
+          app: demo-statefulset
+      serviceName: demo-statefulset
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: demo-statefulset
+        spec:
+          containers:
+          - name: demo-container
+            image: registry.access.redhat.com/ubi8/ubi:latest
+            command: [ "/bin/sh", "-c", "--" ]
+            args: [ "while true; do sleep 30; done;" ]
+            volumeDevices:
+              - name: demo-volume-raw-block
+                devicePath: "/dev/block"
+          volumes:
           - name: demo-volume-raw-block
-            devicePath: "/dev/block"</b>
-      volumes:
-      - name: demo-volume-raw-block
-        persistentVolumeClaim:
-          claimName: demo-pvc-raw-block
-</pre>
+            persistentVolumeClaim:
+              claimName: demo-pvc-raw-block
 
 ## Creating a StatefulSet with both raw block and file system volumes
 
-Create a StatefulSet yaml file, similar to the following demo-statefulset-combined.yaml file.
-
-<pre>
-kind: StatefulSet
-apiVersion: apps/v1
-metadata:
-  name: demo-statefulset-combined
-spec:
-  selector:
-    matchLabels:
-      app: demo-statefulset
-  serviceName: demo-statefulset
-  replicas: 1
-  template:
+Create a StatefulSet YAML file, similar to the following `demo-statefulset.yaml` file.
+
+In a StatefulSet file with two volumes with different volume modes, it is important to indicate both the `volumeMounts` and `volumeDevices` parameters.
```

Comment:
the number of volumes is not relevant here.
```suggestion
In a StatefulSet file with both volume modes, it is important to indicate both the `volumeMounts` and `volumeDevices` parameters.
```

Reply:
DONE